Frameworks/UIKit/UILabel.mm

```diff
@@ -558,11 +500,14 @@ - (UIBaselineAdjustment)baselineAdjustment {
 }
 
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes NSLineBreakByCharWrapping/NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle
+ are not supported. NSLineBreakByCharWrapping is using NSLineBreakByWordWrapping instead.
+ NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle is using NSLineBreakByTruncatingTail instead
```

Comment:
nit: should we log UNIMPLEMENTED_WITH_MSG if any unsupported values are used here?